pandas/tests/series/test_analytics.py

```diff
@@ -895,28 +895,30 @@ def test_count(self):
         ts.iloc[[0, 3, 5]] = nan
         assert_series_equal(ts.count(level=1), right - 1)
 
-    def test_dot(self):
+    @pytest.mark.parametrize('dot_fn', [Series.dot, Series.__matmul__])
```

Comment:
Prefer `operator.matmul` (which corresponds to the `@` operator) to `Series.__matmul__`.